client/src/components/Pages/Create/index.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from "react";
+import { useHistory } from 'react-router-dom';
+import Spinner from "../../Spinner";
+import "./create.css";
+
+export const Create: React.FunctionComponent = () =>{
+    const [pollType, setPollType] = useState("PassFail");
+    const [pollTitle, setPollTitle] = useState("");
+    const [pollOptions, setPollOptions] = useState(["Pass", "Fail or Conditional", "Abstain"]);
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState(false);
+    let history = useHistory();
+
+    function handleSelectPollType(changeEvent:React.FormEvent<HTMLInputElement>) {
+        setPollType(changeEvent.currentTarget.value);
+        switch(changeEvent.currentTarget.value) {
+            case "PassFail":
+                setPollOptions(["Pass", "Fail or Conditional", "Abstain"])
+                break;
+            case "FailConditional":
+                setPollOptions(["Conditional", "Fail", "Abstain"])
+                break;
+            case "Conditional":
+                setPollOptions([""])
+                break;
+            default:
+        }
+    }
+
+    function handleChangePollTitle(changeEvent:React.FormEvent<HTMLInputElement>) {
+        setPollTitle(changeEvent.currentTarget.value);
+    }
+    function handleChangeOption(changeEvent:React.FormEvent<HTMLInputElement>, idx:number) {
+        const newPollOptions = [...pollOptions]
+        newPollOptions[idx] = changeEvent.currentTarget.value;
+        setPollOptions(newPollOptions);
+    }
+
+    function addOption() {
+        const newPollOptions = [...pollOptions]
+        newPollOptions.push("");
+        setPollOptions(newPollOptions);
+    }
+
+    function removeOption(idx:number) {
+        const newPollOptions = [...pollOptions]
+        newPollOptions.splice(idx,1);
+        setPollOptions(newPollOptions);
+    }
+
+      function buttonClick() {
+          setLoading(true);
+          const body = {
+              "title": pollTitle,
+              "options": pollOptions
+          }
+          fetch("http://localhost:5000/api/initializePoll", {
```

Comment:
Have you registered an issue for remapping all these sort of calls away from a hardcoded host and port?

Reply:
It's mentioned in issue #20 - I can make a separate issue specifically for that if we'd prefer though